alot/db/message.py

```diff
@@ -256,7 +261,17 @@ def accumulate_body(self):
         returns bodystring extracted from this mail
         """
         # TODO: allow toggle commands to decide which part is considered body
-        return extract_body(self.get_email())
+        eml = self.get_email()
+        bodytext = extract_body(eml)
+
+        # check if extracted body is empty but msg contains html parts
+        if not bodytext:
+            types = []
+            for part in eml.walk():
+                types.append(part.get_content_type())
+            if 'text/html' in types:
+                bodytext = MISSING_HTML_MSG
+        return bodytext
```

Comment:
Can this be simplified to this:
```py
if not bodytext:
    for part in eml.walk():
        if part.get_content_type() == 'text/html':
            return MISSING_HTML_MSG
return bodytext
```

Reply:
Or even
```py
if not bodytext and 'text/html' in (part.get_content_type() for part in eml.walk()):
    return MISSING_HTML_MSG
return bodytext
```

Reply:
sure, sounds good. I've take the latter and indented the long conditional.